btree/btree.c

```diff
@@ -6956,11 +6951,12 @@ static void ut_multi_stream_kv_oper(void)
 			rc = m0_be_tx_open_sync(tx);
 			M0_ASSERT(rc == 0);
 
-			M0_BTREE_OP_SYNC_WITH_RC(&kv_op,
-						 m0_btree_del(tree,
-							      &del_key_in_tree,
-							      &ut_cb, 0,
-							      &kv_op, tx));
+			rc = M0_BTREE_OP_SYNC_WITH_RC(&kv_op,
+						      m0_btree_del(tree,
+								   &del_key_in_tree,
```

Comment:
80 char limit?

Reply:
I can not change the name of variable, this ut is written by Shashank.

Reply:
No issues, you can bring the statement after rc  = on the below line starting at same point as rc or maybe with one space.
This way we wont exceed the 80 char limit.

Reply:
ok